src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -136,6 +136,7 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
     this.route.data.pipe(
         tap((data: any) => {
           this.intent = data.intent;
+          this.entityType = data.intent === Intent.INSTRUCTOR_SUBMISSION ? 'instructor' : '';
```

Comment:
I really don't think this is a good idea. Only if `data.intent` matches then `entityType` should be changed, but otherwise it should be untouched instead of forced to empty string.

Reply:
Ah yes